src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -22,6 +8,8 @@
 
 public class AdminServlet extends HttpServlet {
 
+/** Directs user to admin page.
+ */
```

Comment:
one line comment, no need to be two lines here.

/** Directs user to admin page. */